.circleci/config.yml

```
@@ -0,0 +1,65 @@
+version: 2.1
+
+references:
+  workdir: &workdir
+    working_directory: ~/source
+
+  persist-workspace: &persist-workspace
+    persist_to_workspace:
+      root: ~/source
+      paths:
+        - .
+
+  attach-workspace: &attach-workspace
+    attach_workspace:
+      at: .
+
+jobs:
+  build:
```

avoid name a job with the same name of the workflow it can confuse circleci sometimes.
```suggestion
  install:
```